sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -4,8 +4,10 @@
 #nullable enable
 
 using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
 using System.Linq;
-using System.IO;
+using System.Reflection;
```

Comment:
This seems like it may be a concern.   @m-redding:  Do you know what the impact would be for AOT compatibility of Core?

Reply:
removed